src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -1526,6 +1529,27 @@ If the toggle is off custom packages that are not already loaded will load once
   <data name="PackageUploadStateUploading" xml:space="preserve">
     <value>Uploading...</value>
   </data>
+  <data name="PackageFilterByHost" xml:space="preserve">
+    <value>HOST</value>
+  </data>
+  <data name="PackageFilterByStatus" xml:space="preserve">
+    <value>STATUS</value>
+  </data>
+  <data name="PackageFilterByDependency" xml:space="preserve">
+    <value>DEPENDENCY</value>
+  </data>
+  <data name="PackageSearchViewContextMenuFilterActive" xml:space="preserve">
+    <value>Active</value>
+  </data>
+  <data name="PackageSearchViewContextMenuFilterDeprecated" xml:space="preserve">
+    <value>Deprecated</value>
+  </data>
+  <data name="PackageSearchViewContextMenuFilterDependencies" xml:space="preserve">
+    <value>Has dependecies</value>
+  </data>
+  <data name="PackageSearchViewContextMenuFilterNoDependencies" xml:space="preserve">
+    <value>Has no dependecies</value>
```

Comment:
I forgot out conclusion, didn't we discuss that we only need this only option with chech/not checked status?

Reply:
Right! I think I might not have understood correctly then, sorry about that. 
So, we only have one filter, which is for packages with `dependencies`. .. But would it work like that? Because we display all packages, with and without dependencies. We can then filter packages with, and packages without. Sorry, if I flip it around - if we only have a filter for 'dependencies', when we activate it, we will show only packages with dependencies. If it is not activated, we will be showing 'dependencies' + 'no dependencies'. But we will never be able to show only 'no dependencies'. 
Should we have another rehash on that to make sure we are on the same page?

Reply:
Users should be able to filter and see only packages that have no dependencies. So if no filter applied, show all packages (with or without dependencies), and when the "Has no dependencies" filter is applied, show only packages with no dependencies.

Reply:
@dnenov Let us know if this make sense

Reply:
Implemented as discussed on the last meeting.